website/docs/gcp_bigquery.md

```diff
@@ -17,18 +34,22 @@ setting sync tool class. A few BigQuery-specific configurations are required.
 | `hoodie.gcp.bigquery.sync.project_id`        | The target Google Cloud project                                                                                 |
 | `hoodie.gcp.bigquery.sync.dataset_name`      | BigQuery dataset name; create before running the sync tool                                                      |
 | `hoodie.gcp.bigquery.sync.dataset_location`  | Region info of the dataset; same as the GCS bucket that stores the Hudi table                                   |
-| `hoodie.gcp.bigquery.sync.source_uri`        | A wildcard path pattern pointing to the first level partition; make sure to include the partition key.          |
+| `hoodie.gcp.bigquery.sync.source_uri`        | A wildcard path pattern pointing to the first level partition; partition key can be specified or auto-inferred. Only required for partitioned tables |
 | `hoodie.gcp.bigquery.sync.source_uri_prefix` | The common prefix of the `source_uri`, usually it's the path to the Hudi table, trailing slash does not matter. |
 | `hoodie.gcp.bigquery.sync.base_path`         | The usual basepath config for Hudi table.                                                                       |
+| `hoodie.gcp.bigquery.sync.use_bq_manifest_file` | Set to true to enable the manifest based sync                                                                |
 
 Refer to `org.apache.hudi.gcp.bigquery.BigQuerySyncConfig` for the complete configuration list.
-
-In addition to the BigQuery-specific configs, set the following Hudi configs to write the Hudi table in the desired way.
+### Partition Handling
+In addition to the BigQuery-specific configs, you will need to use hive style partitioning for partition pruning in BigQuery. On top of that, the value in partition path will be the value returned for that field in your query. For example if you partition on a time-millis field, `time`, with an output format of `time=yyyy-MM-dd`, the query will return `time` values with day level granularity instead of the original milliseconds so keep this in mind while setting up your tables.
 
 ```
 hoodie.datasource.write.hive_style_partitioning = 'true'
+```
+
+For the view based sync you must also specify the following for partitioned tables to drop the value from your base file:
+```
 hoodie.datasource.write.drop.partition.columns  = 'true'
-hoodie.partition.metafile.use.base.format       = 'true'
```

Comment:
I don't see this option in the hudi docs so I removed it. Also we don't use this internally when running the sync tool. Anyone know what this may have been for?

Reply:
yeah. each hudi partition has a metadata file called .hoodie_partition_metadata. this has to be in parquet format just for BQ sync purpose. for other query engines or metastores, we did not have this requirement. 
So, this meta file will be just a text file in general. but if user prefers to sync to BQ, the recommendation is to enable this